templates/look-and-feel/components/check-your-answers-v2.njk

```diff
@@ -0,0 +1,25 @@
+{% macro answer(question, answer, url, id) %}
+<tr class="govuk-table__row" id="cya-{{ safeId(id, question) }}">
+  <th class="govuk-table__header cya-question" scope="row">{{ question }}</th>
```

Comment:
using class `govuk-table__header` will cause this content to be bold ( something i am about to change on master )

Reply:
yep I will change that. It is quite interesting that I tested that on RFE and it doesn't go bold as I expected. I don't know whether it's due to the version of the Gov.uk framework. It would be nice if at some point we can also update the version of the Gov.uk framework to the latest one on this project, that would reduce the need for our own css stylesheets.

Reply:
we can do the upgrade as part of another tech debt when this one is across the line.

not sure why you didn't see bold... `govuk-table__header` class defiantly makes it bold.

Reply:
Yep it should do. Just fixed that :)

Reply:
Also, because of the naming of the css added (i.e. govuk-check-your-answers-v2), no other files in other applications should be affected ... unless one file uses that class name, but I don't think it is the case.